consensus/istanbul/backend/backend.go

```diff
@@ -220,20 +222,20 @@ func (sb *Backend) Gossip(valSet istanbul.ValidatorSet, payload []byte, msgCode
 	return nil
 }
 
+// Enode for backend
 func (sb *Backend) Enode() *enode.Node {
 	if sb.broadcaster != nil {
 		return sb.broadcaster.GetLocalNode()
-	} else {
-		return nil
 	}
+	return nil
 }
 
+// GetNodeKey gets the Node PrivateKey
```

Comment:
nit: I would be careful here to make it clear that this returns the nodekey, i.e. the key used to encrypt messages in the p2p layer, and not, for example, the private key used for signing blcoks